src/Pages/Home/Home.js

```diff
@@ -0,0 +1,384 @@
+import searchIcon from './images/search_icon.png';
+import slideImage1 from './images/slide_img1.jpeg';
+import appBannerImg from './images/phone_banner.png';
+import wecodeLogo from './images/wecode.jpeg';
+import selectIcon from './images/select.png';
+import banana from './images/banana.jpeg';
+import fusion from './images/fusion.jpeg';
+import pear from './images/pear.jpeg';
+import Salad from './images/Salad.jpeg';
+
+const Home = () => {
+  const [tabIndex, setTabIndex] = useState(0);
+  const [bannerIndex, setBannerIndex] = useState(0);
+  const BannerImages = [
+    { target: banana },
+    { target: fusion },
+    { target: pear },
+    { target: Salad },
+  ];
+  const handleSelctType = (e, index) => {
+    setTabIndex(index);
+  };
+
+  useEffect(() => {
+    setTimeout(() => {
+      const target =
+        bannerIndex >= BannerImages.length - 1 ? 0 : bannerIndex + 1;
+      setBannerIndex(target);
+    }, 4000);
+  }, [BannerImages]);
+
+  return (
+    <HomeMainWrap>
+      <BannerBox activeBanner={BannerImages[bannerIndex].target}>
+        <BlurBox />
+        <SearchContainer>
+          <SearchTitle>어떤 맛집을 찾고 계세요?</SearchTitle>
+          <SelectBox>
+            {selectType.map((select, index) => {
+              return (
+                <SelectItem
+                  key={index}
+                  onClick={e => {
+                    handleSelctType(e, index);
+                  }}
+                >
+                  <SelectTargetText activeCheck={tabIndex === index}>
+                    {select.title}
+                  </SelectTargetText>
+                </SelectItem>
+              );
+            })}
+            <SelectTargetIcon tabIndex={tabIndex}></SelectTargetIcon>
+          </SelectBox>
+          <SearchInputBox>
+            <SearchInput
+              type="text"
+              placeholder="원하시는 지역명, 지하철역, 가게명을 입력해주세요"
+            />
+            <SearchBtn>
+              <SearchBtnIcon />
+            </SearchBtn>
+          </SearchInputBox>
+        </SearchContainer>
+      </BannerBox>
+      <SectionBox>
+        <NoticeBox>
+          {noticeData.map((notice, index) => {
+            const leftSpace = index === 0;
+            return (
+              <NoticeItem key={index} leftSpace={leftSpace}>
+                <NoticeHeader>{notice.title}</NoticeHeader>
+                <NoticeContent>
+                  {index === 0 ? (
+                    <NoticeImage src={slideImage1} alt="광고 이미지" />
+                  ) : (
+                    <NoticeListBox>
+                      {notice.sub.map((text, index) => (
+                        <NoticeListItem key={index}>{text}</NoticeListItem>
+                      ))}
+                    </NoticeListBox>
+                  )}
+                </NoticeContent>
+              </NoticeItem>
+            );
+          })}
+        </NoticeBox>
+      </SectionBox>
+      <SectionBox odd={true}>
```

Comment:
true 값으로 전달되는 props는 ={true} 선언 없이 그냥 props이름만 적어주셔도 됩니다
```suggestion
      <SectionBox odd>
```

Reply:
오! 꿀팁 감사합니다!